web_poet/page_inputs/http.py

```diff
@@ -261,6 +263,10 @@ def urljoin(self, url: Union[str, _RequestUrl, _ResponseUrl]) -> _RequestUrl:
         *self*."""
         return _RequestUrl(urljoin(self._base_url, str(url)))
 
+    @memoizemethod_noargs
+    def _bom_encoding(self):
+        return read_bom(self.body)[0]
```

Comment:
I wonder if this code should be moved to HttpResponseBody class

Reply:
Good call, moved it in https://github.com/scrapinghub/web-poet/pull/81/commits/c39d176050229d4ccb3bbf639ad252e5943fa23e.